index.ts

```diff
@@ -129,9 +129,6 @@ export {getQueryParams} from './helpers/url';
 export {join as joinUrl} from './helpers/url';
 export {withQueryParams} from './helpers/url';
 
-// Schema helpers.
-export {getEffectivePropertyKeysFromSchema} from './helpers/schema';
```

Comment:
@huayang-codaio I think you meant this one to be exposed to makers, so it should probably stay here, right?

Reply:
yeah this intentionally stays here.